raster-foundry/raster-foundry
# Adjust histogram / breakpoints behavior

## Overview

Visible range on histogram contains all histogram datapoints and the entire range covered by breakpoints
Instead of hiding breakpoints when location is < 0, set to 0
Fix reverse button in color scheme dropdown 


### Checklist

~- [ ] Styleguide updated, if necessary~
~- [ ] Swagger specification updated, if necessary~
~- [ ] Symlinks from new migrations present or corrected for any new migrations~

## Testing Instructions

 * Verify that histogram breakpoints no longer randomly get hidden for landsat scenes
* Verify that the histogram view range works as described
* Verify that changing color schemes does not change the location of breakpoints or the view range of the histogram

Closes https://github.com/raster-foundry/raster-foundry/issues/2706
